### DISCUSSION TITLE
This is the best Offline LLM!

### DISCUSSION THREAD

**Discussion by alxspiker** [Mobile-Artificial-Intelligence/maid#54] [General]
Plan to help out with the project starting tomorrow, first thing I am going to do is work on the model and character import/switching settings. Very buggy.

The only thing I dislike Maid over Sherpa is that the ggml model I have has no safe tensors. Maybe I can find a newer gguf without the safe tensors. But then again, isn't that like a setting you can turn off?

The only thing I personally would want for functionality is a way to ingest documents into a vertex database or something.

P.S. Sorry for all the discussions, just so excited for this as I have been a big Sherpa user since day 1. Spent all day on the charger trying to keep my phone powered and a fan blowing on it to keep it cool.

**Comment by danemadsen** [comment]
Much obliged. GGML models are only supported in the GGML branch which is no longer actively maintained. You can compile and use that branch if you want to use GGML. I dont mind the discussions, create away, but if you find an issue please log it in the issues tab so i know what needs fixing
